include/deal.II/grid/reference_cell.h

```diff
@@ -2364,15 +2318,15 @@ namespace internal
     inline const internal::ReferenceCell::Base &
     get_cell(const dealii::ReferenceCell &type)
     {
-      static const std::array<std::unique_ptr<internal::ReferenceCell::Base>, 8>
-        gei{{std::make_unique<internal::ReferenceCell::Vertex>(),
-             std::make_unique<internal::ReferenceCell::Line>(),
-             std::make_unique<internal::ReferenceCell::Triangle>(),
-             std::make_unique<internal::ReferenceCell::Quadrilateral>(),
-             std::make_unique<internal::ReferenceCell::Tetrahedron>(),
-             std::make_unique<internal::ReferenceCell::Pyramid>(),
-             std::make_unique<internal::ReferenceCell::Wedge>(),
-             std::make_unique<internal::ReferenceCell::Hexahedron>()}};
+      static const std::unique_ptr<internal::ReferenceCell::Base> gei[8] = {
+        std::make_unique<internal::ReferenceCell::Vertex>(),
+        std::make_unique<internal::ReferenceCell::Line>(),
+        std::make_unique<internal::ReferenceCell::Triangle>(),
+        std::make_unique<internal::ReferenceCell::Quadrilateral>(),
+        std::make_unique<internal::ReferenceCell::Tetrahedron>(),
+        std::make_unique<internal::ReferenceCell::Pyramid>(),
+        std::make_unique<internal::ReferenceCell::Wedge>(),
+        std::make_unique<internal::ReferenceCell::Hexahedron>()};
```

Comment:
I do not like this change here. There is really no improved readibility for 
```
static const std::unique_ptr<internal::ReferenceCell::Base> gei[8] = {
```
compare to
```
static const std::array<std::unique_ptr<internal::ReferenceCell::Base>, 8>
```

To the contrary, it is easier to miss the `[8]` than the `std::array`.